index.html

```diff
@@ -13,6 +13,8 @@
     <link rel="stylesheet" href="./Components/Card/card.css">
     <link rel="stylesheet" href="./Components/Input/input.css">
     <link rel="stylesheet" href="./Components/Image/image.css">
+    <link rel="stylesheet" href="./Components/Avatar/avatar.css">
+    <link rel="stylesheet" href="./Components/Badge/badge.css">

```

Comment:
I personally prefer to use all lower case letters in the website files because that is considered a good practice.

Reply:
oh, I just read about it online, and thanks for telling me 🙌🏼